app/overrides/add_paid_to_orders_admin_index.rb

```diff
@@ -0,0 +1,6 @@
+Deface::Override.new(virtual_path: 'spree/admin/orders/index',
+  name: 'add_paid_to_orders_admin_index',
+  # remove: ".nav-item:first")
+  insert_after: ".nav-item:first",
+  text: "<li class=\"nav-item\"><%= link_to 'Paid', params.merge({q: {payment_state_eq: :paid, completed_at_not_null: 1}}).permit!,
```

Comment:
Using JS/jQuery this should be easy

Reply:
https://github.com/TomaszKot11/ror-junior/commit/221c4858fb09da63914f666cd0a6f87c427cb63b - This is brute force